tests/requestsOcsTest.js

```diff
@@ -60,11 +66,11 @@ describe('Main: Currently testing low level OCS', function () {
         headers: validAdminAuthHeaders
       })
       .willRespondWith({
-        status: 401,
+        status: 404,
```

Comment:
shouldn't it be `401`, and the ocs status code be `997` according to the issue that you mentioned in the comment above? :thinking:

Reply:
`401` is not expected while trying to update `non-existing` user by admin

expected for v1: http `200` and ocs `101`
expected for v2: http `404` and ocs `404`